sources/player/Player.gd

```diff
@@ -0,0 +1,39 @@
+extends KinematicBody2D
+class_name Player
+signal interact
+
+export (float) var speed = 400
+
+var movement = Vector2(0, 0)
+
+
```

Comment:
I'm new with group in Godot. But do we need add player to  group actor to link with other interactable entities?

Reply:
Having a node in group actor allows for interactable to check if the given body can interract with them. The alternative would be to create a superclass all actors would inherit from. In GDScript this is not trivial so I went for the group option.

Reply:
Got it. I've read about group before, but forgot it until see you implementation.